projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/smeltery/alloying.json

```
@@ -0,0 +1,16 @@
+{
+  "title": "合金",
+  "text": [
+    {
+      "text": "合金是将较低级材料融合成更为强力的材料的过程。"
+    },
+    {
+      "text": "你需要做的就是找到几种可以合金的材料，将它们注入装填有燃料的冶炼炉，并静待它们融合为全新的强力材料。",
+      "paragraph": true
+    },
+    {
+      "text": "某些材料可能需要燃烧温度更高的燃料才能合金。",
```

```suggestion
      "text": "某些材料可能需要温度更高的燃料才能变成合金。",
```
语意重复。“需要燃料”就行了，“燃烧燃料”有点多余。